packages/shopify-app-session-storage-mysql/src/mysql-connection.ts

```diff
@@ -73,6 +73,14 @@ export class MySqlConnection implements RdbmsConnection {
     return decodeURIComponent(this.dbUrl.pathname.slice(1));
   }
 
+  public getSSL(): string | mysql.SslOptions | undefined {
+    const ssl = this.dbUrl.searchParams.get('ssl');
+    if (!ssl) {
+      return undefined;
+    }
+    return JSON.parse(ssl);
```

Comment:
I would just do this:
```javascript
return ssl ? JSON.parse(ssl) : undefined;  
```

Reply:
@maruffahmed agreed my preference is also shorthand, updated.